common/azure-common/src/test/java/com/azure/common/implementation/util/FluxUtilTests.java

```diff
@@ -251,4 +265,19 @@ public void toByteArrayWithNonEmptyByteBuffer() {
         return bytes;
     }
 
+    /**
+     * Helper class for creating file if not exists
+     *
+     * @param fileName
+     * @return File file
+     * @throws IOException
+     */
```

Comment:
nit; missing doc for params. This is (1). a private method & (2). function name explain what it does, so it is not necessary to have a doc.

Reply:
Removed.